R/load_containers.R

```diff
@@ -0,0 +1,71 @@
+#' Return blob container
+#'
+#' Create endpoint URL to access Azure blob or file storage on either the
+#' `dev` or `prod` stage from specified storage account
+#' @param containers `character` vector containing the name of container to
+#'     load (default, "global", "projects")
+#' @param sas_key Shared access signature key to access the storage account or
+#'    blob. Default is set to use  dev stage sas key via an a env var named
+#'    "DSCI_AZ_SAS_DEV"
+#' @param service Service to access, either `blob` (default) or `file.`
+#' @param stage Store to access, either `prod` (default) or `dev`. `dev`
+#' @param storage_account Storage account to access. Default is `imb0chd0`
+#' @examples
+#' # load project containers
+#' pc <- load_containers(containers = "projects")
+#' AzureStor::list_blobs(
+#'   container = pc$PROJECTS_CONT,
+#'   dir = "ds-contingency-pak-floods"
+#' )
+#'
+#' # You can also list as many containers as you want.
+#' pc <- load_containers(containers = c("global", "projects"))
+#' AzureStor::list_blobs(
+#'   container = pc$GLOBAL_CONT,
+#'   dir = "raster/cogs"
+#' )
+#' @export
+load_containers <- function(
+    containers = c("global", "projects"),
```

Comment:
would be kind of cool if we could use:

```
AzureStor::list_blob_containers()
```

in the event that no containers are specified as well as for checking container args. However it seems somehow our blob permissions do not allow this. I guess if we specify a container that doesn't exist we will get the appropriate warning anyways from `{AzureStor}` so not really critical for checking args